app/src/main/java/de/danoeh/antennapod/activity/VideoplayerActivity.java

```diff
@@ -309,12 +309,14 @@ protected void onReloadNotification(int notificationCode) {
     public void onStartTrackingTouch(SeekBar seekBar) {
         super.onStartTrackingTouch(seekBar);
         videoControlsHider.stop();
+        controls.setVisibility(View.GONE);
```

Comment:
Is this needed? I think it calls the super function that already does that.

Reply:
Here's what it looks like without that line, my goal is just to hide the three main buttons while scrubbing like it does in newpipe. I welcome an opinion on this.
![Screenshot_1582845334](https://user-images.githubusercontent.com/49007921/75495779-a0fd6800-597c-11ea-9e9b-10d084b77834.png)

Reply:
Adding a screenshot with the line for comparison.
![Screenshot_1582846654](https://user-images.githubusercontent.com/49007921/75496783-85e02780-597f-11ea-80eb-9ee1a2711e7e.png)

Reply:
Oh, okay. I misread the line and thought you are setting the position overlay visibility again.